### ISSUE TITLE
TODO : Add a mini setup script that downloads all the code.

### ISSUE THREAD

**Issue by lowki** [wakdev/slash-cms#61]
Would be great to avoid downloading all sources then uploading all again to the server.

**Comment by wakdev** [comment]
Good idea ! And i have already an address for downloading : http://packages.slash-cms.com/releases/ 
We can imagine several channel update (release, beta, alpha ...) but I think there are more important things first ^^
